pkgs/stdenv/generic/check-meta.nix

```diff
@@ -439,6 +450,18 @@ let
     # Please also update the type in /pkgs/top-level/config.nix alongside this.
     else if hasNoMaintainers attrs then
       { valid = "warn"; reason = "maintainerless"; errormsg = "has no maintainers"; }
+    else if unstructuredArrays attrs != [] then
+      let
+        quote = s: ''"${s}"'';
+        variables = unstructuredArrays attrs;
+        variablesStr = builtins.concatStringsSep ", " (map quote variables);
+        usePlural = variables != [(builtins.head variables)];
+        plural = optionalString usePlural "s";
```

Comment:
This is pretty cute.

Reply:
Is there a reason we’re spelling `lib.length variables != 1` like that…?